Apps/Contoso.Forms.Puppet/Contoso.Forms.Puppet.iOS/Contoso.Forms.Puppet.iOS.csproj

```diff
@@ -1,168 +1,168 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Do you need that change? Is that the current formatter rule or is it caused by a merge?